wercker.yml

```diff
@@ -9,16 +9,16 @@ build:
         code: ./build
 
     - script:
-        name: copy binary
-        code: cp bin/linux_amd64/farva-gateway "$WERCKER_OUTPUT_DIR"
+        name: copy binaries
+        code: cp bin/linux_amd64/* "$WERCKER_OUTPUT_DIR"
 
 deploy:
     box: nginx:1.10.0
     steps:
     - internal/docker-push:
         username: $QUAY_USERNAME
         password: $QUAY_PASSWORD
-        cmd: /pipeline/source/farva-gateway
+        working-dir: /pipeline/source/
```

Comment:
It wasn't changed back. Take a look at master and let me know if you still want to talk.